### ISSUE TITLE
Document How To Fix Violations in .editorconfig

### ISSUE THREAD

**Issue by RealDotNetDave** [DotNetAnalyzers/StyleCopAnalyzers#3226]
I was just trying to figure out how to turn off this warning: SA1309FieldNamesMustNotBeginWithUnderscore in the .editorconfig. Along with the attribute you documented, it would be helpful if how to setup it up in the .editorconfig file.

**Comment by xmarshal** [comment]
how StyleCopAnalyzers operate with .editorconfig, should we use ruleset instead

**Comment by manfred-brands** [comment]
You can add lines like the below to .editorconfig:
```
# Field names should not begin with underscore
dotnet_diagnostic.SA1309.severity = none
```

**Comment by RealDotNetDave** [comment]
Ok, but it would be nice if that was documented on the SA pages.

**Comment by manfred-brands** [comment]
It is not SA specific, but standard analyzer [configuration](https://docs.microsoft.com/en-us/visualstudio/code-quality/use-roslyn-analyzers?view=vs-2019#set-rule-severity-in-an-editorconfig-file)
You can also do this from within visual studio by hovering over the error and select suppress:

![image](https://user-images.githubusercontent.com/8655218/102152374-d6455900-3eaf-11eb-98e5-46f4300e0a1c.png)

**Comment by RealDotNetDave** [comment]
Can you see how developers, even me are confused? We are told to use the .editorConfig, which I setup. But now, you are saying I need to add the ignore? What is the point of the .editorConfig if it's being ignored?

**Comment by manfred-brands** [comment]
> you are saying I need to add the ignore

What are you referring to? You asked how to disable a rule and I showed you how. The .editorconfig is used.
